include/IDriver.h

```diff
@@ -278,7 +278,7 @@ namespace video
             }
 
             template<typename AssetType>
-            core::vector<core::smart_refctd_ptr<typename video::asset_traits<AssetType>::GPUObjectType> > getGPUObjectsFromAssets(AssetType* const* const _begin, AssetType* const* const _end, IGPUObjectFromAssetConverter* _converter = nullptr);
+            created_gpu_object_array<AssetType> getGPUObjectsFromAssets(AssetType* const* const _begin, AssetType* const* const _end, IGPUObjectFromAssetConverter* _converter = nullptr);
```

Comment:
@Crisspl familiarize yourself with the changes in this commit

Reply:
which commit is this actually?

Reply:
github doesn't show me

Reply:
https://github.com/Crisspl/IrrlichtBAW/pull/10/commits/2c8f3150edcaa5faa7c7fd957ef871fb3e3b164c